chall02/krusthol.c

```diff
@@ -0,0 +1,43 @@
+#include <stdlib.h>
+
+static void	convert_hex(char *hex_table, char *hex_value, int value, int loc)
+{
+	if (value > 15)
+	{
+		hex_value[loc + 1] = hex_table[value % 16];
+		hex_value[loc] = hex_table[(value /= 16)];
+	}
+	else
+	{
+		hex_value[loc] = '0';
+		hex_value[loc + 1] = hex_table[value];
+	}
+}
+
+static char	*error_range(char *hex_value)
+{
+	hex_value[1] = 'e';
+	hex_value[2] = 'r';
+	hex_value[3] = 'r';
+	hex_value[4] = 'o';
+	hex_value[5] = 'r';
+	hex_value[6] = '!';
+	return (hex_value);
+}
```

Comment:
ahahah :)
nice, but no if there is an input error never send back this, return NULL is better for the other function that would use hv_rgb2hex to check if ret is null or no (like malloc do)